# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../alsa-tools/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual label end of ... statement for gcc > 3.4.x.

  - Rene Rebe <rene@rocklinux-consulting.de>

--- alsa-tools-1.0.6/as10k1/as10k1.c	2003-07-15 20:33:59.000000000 +0200
+++ alsa-tools-1.0.6-fixed/as10k1/as10k1.c	2004-09-14 17:11:08.451196236 +0200
@@ -159,6 +159,7 @@
 
 		}
 next_cli_arg:
+		;
 
 	}
 }
